src/app/features/fss-search/fss-search.component.html

```diff
@@ -1,9 +1,13 @@
-<div class="spinner-wrapper" *ngIf="displayLoader"><i class="fas fa-spinner fa-pulse fa-3x"></i></div>
+<div class="spinner-wrapper" *ngIf="displayLoader" aria-live='polite' role="alert" aria-label="results loding"><i class="fas fa-spinner fa-pulse fa-3x" aria-hidden="true"></i></div>
```

Comment:
Assume this should read aria-label="Results Loading"

Capitalise and correct spelling.

Not sure about the capitalisation but noticed "Download File" is capitalised elsewhere (previous file) so we should be consistent.

Reply:
Resolve Typo as commented .

Thanks